testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestComments.java

```diff
@@ -142,10 +142,12 @@ public void testCommentColumn()
                         "CREATE TABLE hive.default.%s (\n" +
                         "   c1 bigint COMMENT 'test comment',\n" +
                         "   c2 bigint COMMENT '',\n" +
-                        "   c3 bigint\n" +
+                        "   c3 bigint,\n" +
+                        "   c4 bigint COMMENT 'test partition comment'\n" +
```

Comment:
nit/optional: Let's name `c4` as `part` / `part_col`

Reply:
I prefer to keep it in sync with already existing code.